genesis_ledgers/devnet.json

```diff
@@ -18,12 +18,12 @@
     "staking": {
       "seed": "2vbdR3vZUxh8NFnsnrefko7YCiLUKNPUbQpqSQAFXFijtTjd3531",
       "hash": "jxvdSnpgWjjERJFdcYNxChJ9F39v4tGrNSzDcaQXuXeJnicrYEP",
-      "s3_data_hash": "84a349f6dd12c76b694b283975885994bcfb9117c3f41fa941e0229da3ce2a33"
+      "s3_data_hash": "5a83ae9a875a2e1c5dd433c28a4a94b4db2064ae36e6bf1e272821afc33e2315"
```

Comment:
I don't think this change should exist.
Why we can't keep the old hash by importing the necessary file to a upload-only bucket?

Reply:
this was the ledger hash when i uploaded to the read only bucket so i kept it in this pr as well. because it was possible to connect to the network, sync, and obtain peers i saw no issues in changing the hash.

Reply:
It's better to keep the old hash
Changing the hash means that we need to release another README/tooling to community that will prove this new hash is legit

We need to re-upload files to the `ro` bucket, it seems so

Reply:
Ok we can do that as long as the ledger hashes match I think